xAPI.md

```diff
@@ -507,13 +507,39 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 ## 7.0 xAPI Components
 
-## 8.0 Getting More Out of xAPI
 
-### 8.1 Extending xAPI
 
-### 8.2 Profiles
+## 8.0 Extending xAPI
+
+xAPI can be extended in a few ways.  The most notable and extensive is Statement Extensions, which allow great flexibility within Statements.  It is recommended that profiles or Communities or Practice agree on how to use  
+extensions for their particular use cases. Implementation details are covered in a [later section](#miscext)
+
+The About Resource is another place xAPI supports Extensions.  The LRS may find it useful to communicate features or behaviors beyond this specification to activity provider.  It is hoped that if more information is available about an LRS, that the About Resource contains it.
+
+Finally, the set of Resources implemented is not expected to be constrained by this document. Resources beyond 
+those listed in this specification can be implemented and co-exist with current Resources.
```

Comment:
I'm not keen on "current Resources"; how about "the resources defined in this specification"?